### PR TITLE
Chore: Create new dimension always_exclude for exclusions which applies to all production data analyses

### PR DESCRIPTION
**Pull request by sri-byte**
Impact: Creates a new dimension `company_scorecard_exclusions` for exclusions which commonly apply to all production data analyses.

### CODE REVIEW

**Review by catalintomai — APPROVED** [review]

File: views/marts/product/dim_excludable_servers.view.lkml [hunk 1]
```diff
@@ -136,4 +136,12 @@ view: dim_excludable_servers {
     view_label: "Server: Exclusion Reasons"
   }
 
+  dimension: always_exclude {
+    type: yesno
+    sql:  ${has_reason_single_day_security_only} or ${has_reason_custom_build_version_format} or ${has_reason_single_day_server_side_telemetry_only} or ${has_reason_single_day_telemetry_only} or ${has_reason_ran_tests} or ${has_reason_single_day_user_telemetry_only} or ${has_reason_restricted_ip} or ${has_reason_invalid_server_id} or ${has_reason_internal_email} or ${has_reason_test_server} or ${has_reason_community};;
```
Comment (ifoukarakis) [inline views/marts/product/dim_excludable_servers.view.lkml:141]: One important thing is that the table contains only the servers with exclusion reasons, and not the full list of servers. So this dimension **MUST** be used always with a `not` clause.

An improvement on the original dimension table would be to include all servers, even those without any exclusion reasons.

Comment (ifoukarakis) [inline views/marts/product/dim_excludable_servers.view.lkml:141]: Are all these needed? The common subset so far doesn't include the single day telemetry/security ones.

File: views/marts/product/dim_excludable_servers.view.lkml [hunk 2]
```diff
@@ -136,4 +136,12 @@ view: dim_excludable_servers {
...
+    label: "Always Exclude"
+    description: "A combination of exclusion reasons which applies to production data analyses"
```
Comment (ifoukarakis) [inline views/marts/product/dim_excludable_servers.view.lkml:143]: The label and description are too vague. Let's find a more representative name for this group of exclusion reasons.

**Review by ifoukarakis — CHANGES_REQUESTED** [review]

File: views/marts/product/dim_excludable_servers.view.lkml [hunk 3]
```diff
@@ -136,4 +136,12 @@ view: dim_excludable_servers {
     view_label: "Server: Exclusion Reasons"
   }
 
+  dimension: company_scorecard_exclusions {
+    type: yesno
+    sql: ${has_reason_active_users__registered_users} or ${has_reason_community} ${has_reason_invalid_server_id} or ${has_reason_internal_email} or ${has_reason_custom_build_version_format} or ${has_reason_no_stripe_installation_found} or ${has_reason_ran_tests} or ${has_reason_restricted_ip} or ${has_reason_test_server};;
```
Comment (catalintomai) [inline views/marts/product/dim_excludable_servers.view.lkml:141]: missing "or" between this and next?

Comment (sri-byte) [reply views/marts/product/dim_excludable_servers.view.lkml]: Fixed this.

**Review by catalintomai — APPROVED** [review]

**Review by ifoukarakis — APPROVED** [review]

**Pull request merged**